config/vufind/sitemap.ini

```diff
@@ -43,11 +43,11 @@ fileLocation   = /tmp
 index[] = "Solr,/Record/"
 ;index[] = "SolrAuth,/Authority/Record?id="
 
-;  This setting controls how IDs are retrieved from the index. It may be 'terms'
-;    (the default, and the faster option) to use the terms component, or 'search'
-;    to retrieve results using a normal search (which is useful if you need to
-;    apply hidden filters to your results, or if you do not have terms enabled).
-retrievalMode = terms
+;  This setting controls how IDs are retrieved from the index. It may be
+;  'cursorMark' (the default, most compatible but slower method), or 'terms' (the
+;    faster option). Note that 'terms' method does not support hidden filters or
+;    other limiting options and requires that the index has terms enabled.
+retrievalMode = cursorMark
```

Comment:
Is there a strong reason to replace the 'search' mode with 'cursorMark'? Even though you are using a cursor mark, it is still a search-based retrieval method, so I don't think renaming it (and thus potentially breaking existing search-based configurations) is necessary. Let me know if you feel otherwise!